docs/hardware.rst

```diff
@@ -82,10 +82,10 @@ Workstations
   <https://tails.boum.org/news/Tails_3.0_will_require_a_64-bit_processor/index.en.html>`_.
 
 These components are necessary to do the initial installation of
-SecureDrop and to process submissions using the airgapped workflow.
+SecureDrop and to process submissions using the air-gapped workflow.
```

Comment:
That's fair. These stylistic changes -- especially when they stretch across the entire document set -- are better served through their own scoped PRs.